src/main/java/io/apicurio/datamodels/core/validation/rules/required/AaMissingCorrelationIdRule.java

```diff
@@ -0,0 +1,17 @@
+package io.apicurio.datamodels.core.validation.rules.required;
```

Comment:
We need to make sure we add the standard file header comment to all new java files you've created.  Please see the header found in the existing classes and copy/paste that to all new .java files.

Reply:
done